tests/test_ext_commands.py

```diff
@@ -44,20 +45,38 @@ def test_sanitize_valid_file(self, sanitizer_config, tmpdir):
         assert outfile.read_text(encoding="utf8") == out_text
 
     def test_removes_file_with_shred_marker(self, sanitizer_config, fake_repo):
-        """Test that sanitize-repo does not send any files that contain a shred
+        """Test that sanitize-file does not send any files that contain a shred
```

Comment:
This comment doesn't make sense, sanitize-file has no target branch. This is not what this test actually tests.